packages/lit-dev-tools-cjs/src/api-docs/generate.ts

```diff
@@ -89,37 +89,42 @@ const analyze = async (config: ApiDocsConfig) => {
   }
 
   console.log(`Analyzing ${config.gitDir}`);
-  const app = new typedoc.Application();
-  app.options.addReader(new typedoc.TSConfigReader());
-  app.bootstrap({
-    tsconfig: config.tsConfigPath,
-    entryPoints: config.entrypointModules,
-    entryPointStrategy: typedoc.EntryPointStrategy.Expand,
-  });
-  const root = app.convert();
-  if (!root) {
-    throw new Error('TypeDoc.Application.convert() returned undefined');
-  }
+  const allPages: Pages = [];
+  const allSymbols: SymbolMap = {};
+  for (const pkg of config.packages) {
+    const app = new typedoc.Application();
```

Comment:
At some point I thought I read some docs that typedoc had options for a monorepo, so you didn't need to make a new app for each package. I can't remember it though. This might matter for getting crosslinks between packages. Right now `ReactiveElement` say isn't a link in the context docs.

Reply:
Yeah - I need to investigate this further. I feel like I read that somewhere as well.